.travis.yml

```diff
@@ -1,10 +1,10 @@
-# Copyright Siemens AG, 2014-2018
-# SPDX-License-Identifier:	GPL-2.0 LGPL-2.1
+# Copyright Siemens AG, 2014-2019
+# SPDX-License-Identifier: GPL-2.0 LGPL-2.1
```

Comment:
This is not a valid SPDX identifier expression. One has to add `AND` or `OR`
```suggestion
# SPDX-License-Identifier: GPL-2.0 OR LGPL-2.1
```

Reply:
right, but it appears being an older issue, which could be excluded from this PR ...